### PR TITLE
Cooking Tweaks

### LINKED ISSUE #7823: Cargo pizzas have no nutriment.
pizza

The upper far right one is a kitchen-made pizza shown for example.

### PR DESCRIPTION
**Pull request by phil235**
Buffs nutriment amounts in doughs and their cooked counterparts, as well as meats.
Fixed a recipe.
Adding renaming custom food with a pen. (e.g. " 'Artery Buster' burger")
Fixes #7823

### CODE REVIEW

File: code/modules/food&drinks/food/customizables.dm [hunk 1]
```diff
@@ -54,6 +54,19 @@
 			update_overlays(S)
 			user << "<span class='notice'>You add the [I.name] to the [name].</span>"
 			update_name(S)
+	else if(istype(I, /obj/item/weapon/pen))
+		var/txt = stripped_input(user, "What would you like the food to be called?", "Food Naming", "", 30)
+		if(txt)
+			ingMax = ingredients.len
+			user << "<span class='notice'>You add a last touch to the dish by renaming it.</span>
```
Comment (optimumtact) [inline code/modules/food&drinks/food/customizables.dm]: No ending " on this line, as pointed out by travis

**Comment by Chocobro** [comment]
>added renaming custom food with a pen

get hype

**Comment by Iamgoofball** [comment]
Finally

**Comment by dannno** [comment]
who cares lmao

**Comment by paprka** [comment]
Iamgoofball wants to force feed people his big dick burger that's all

Seriously though don't do that phil

**Comment by Iamgoofball** [comment]
@dannno people who play chef and don't want their nice name ruined by an appended extra to it?

**Comment by paprka** [comment]
but that's open for a lot of abuse, and on every server where that's happened, people have done pretty much nothing but abuse it by naming their food dishes stupid shit. That was rejected on /tg/, this is a compromise.

**Comment by Cheridan** [comment]
Yes, there's a reason why books are named "Book: Book Title" now.

**Pull request merged**